requirements.txt

```diff
@@ -1,4 +1,8 @@
+datasets
 docstring-parser
 einops
 numpy
-torch
\ No newline at end of file
+seaborn
+tabulate
+torch
+transformers
```

Comment:
Do we need `datasets` and `transformers` directly from the library, or can we prune these?

Reply:
OK, cool, sorry yeah I don't feel too strongly about this & what you describe sounds good, I just have this general "library bias" away from big complex dependencies, where possible.